core/spec/models/spree/promotion_handler/free_shipping_spec.rb

```diff
@@ -13,8 +13,20 @@ module PromotionHandler
       context 'with apply_automatically' do
         let!(:promotion) { create(:promotion, apply_automatically: true, promotion_actions: [action]) }
 
-        it "creates the adjustment" do
-          expect { subject.activate }.to change { shipment.adjustments.count }.by(1)
+        context 'for eligible promotion' do
+          before { allow_any_instance_of(Spree::Promotion).to receive(:eligible?).and_return(true) }
```

Comment:
You don't need this because the default promotion has no rules and is eligible, right?